openedx/adg/lms/applications/constants.py

```diff
@@ -118,3 +118,17 @@
 
 PROGRAM_PREREQ_COURSES_TITLE = f'<strong>{_("Program Prerequisite Courses:")}</strong>'
 BU_PREREQ_COURSES_TITLE = f'<strong>{_("Business Unit Prerequisite Courses:")}</strong>'
+
```

Comment:
Pls remove extra blank line.

Reply:
@ahsanarbisoft please incorporate this suggestion.